src/components/player/index.jsx

```
@@ -0,0 +1,180 @@
+
+  useEffect(() => {
+    getAllLikes();
+    getAllComments();
+  }, [video?.video_id, dispatch]);
+
+  return (
+    <>
+      <Modal title={video?.name} isOpen={isOpen} onClose={onClose} size="lg">
+        {/* Video Player */}
+        <div className="player">
+          <video src={video?.video} controls className="w-full h-full" />
```

Biome (1.9.4)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)